gradle/overrides/login-web-resource-bundle-override/src/main/java/com/liferay/blade/login/web/resource/bundle/override/LoginModuleResourceBundleLoader.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2000-present Liferay, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.liferay.blade.login.web.resource.bundle.override;
+
+import com.liferay.portal.kernel.util.AggregateResourceBundleLoader;
+import com.liferay.portal.kernel.util.CacheResourceBundleLoader;
+import com.liferay.portal.kernel.util.ClassResourceBundleLoader;
+import com.liferay.portal.kernel.util.ResourceBundleLoader;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Liferay
+ */
+@Component(
+	immediate = true,
+	property = {
+		"bundle.symbolic.name=com.liferay.login.web",
+		"resource.bundle.base.name=content.Language",
+		"servlet.context.name=login-web"
+	}
+)
+public class LoginModuleResourceBundleLoader implements ResourceBundleLoader {
+
+	@Override
+	public ResourceBundle loadResourceBundle(Locale locale) {
+		return _resourceBundleLoader.loadResourceBundle(locale);
+	}
+
+	@Reference(
+		target = "(&(bundle.symbolic.name=com.liferay.login.web)(!(resource.bundle.aggregate=*))(!(component.name=com.liferay.samples.login.web.resource.bundle.LoginModuleResourceBundleLoader)))"
```

Comment:
Hey @lawrence-lee  after thinking about this, the only difference between the two samples is this one single line.  My proposal is that we just modify the previous sample to add the `(!(resource.bundle.aggregate=*)) since that is good practice anyways to ensure that the correct resource bundle is referenced.  THen I think we can simply remove the new sample.  WDYT?